megatron/model/activations.py

```diff
@@ -0,0 +1,40 @@
+import torch
+from torch import nn
+from torch.nn import functional as F
+
+
+class _GLUBaseModule(nn.Module):
+    def __init__(self, activation_fn):
+        super().__init__()
+        self.activation_fn = activation_fn
+    
+    def forward(self, x):
+        # dim=-1 breaks in pt<1.10
```

Comment:
```suggestion
        # dim=-1 breaks in jit for pt<1.10
```

sorry, I thought my initial comment was lacking important context that it's a jit issue. fixed now.

Reply:
It's helpful detail, thanks for catching that.